cmd/billing.go

```diff
@@ -0,0 +1,90 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/speakeasy-api/huh"
+	sdkGenConfig "github.com/speakeasy-api/sdk-gen-config"
+	"github.com/speakeasy-api/speakeasy/internal/charm"
+	"github.com/speakeasy-api/speakeasy/internal/charm/styles"
+	"github.com/speakeasy-api/speakeasy/internal/log"
+	"github.com/spf13/cobra"
+)
+
+var billingCmd = &cobra.Command{
```

Comment:
Mind referencing the other commands for how to use our internal `model.ExecutableCommand`? We don't use cobra.Command directly anymore